docs/pbc/all/warehouse-management-system/202311.0/unified-commerce/fulfillment-app-feature-overview.md

```diff
@@ -0,0 +1,112 @@
+
+![screenshot with two picklists]
+
+## Assigning a warehouse
+
+By default, when a customer orders several items of the same SKU, the requested item’s stock is checked in all of the store's warehouses. The warehouses are sorted in the descending order of how much stock of the item they are holding. The warehouse holding the biggest stock of the item is assigned to the order. The warehouse holding the never-out-of-stock item quantity is considered to be holding the biggest stock.
+
+
+In some cases, the ordered quantity might not be available in the warehouse holding the biggest stock of the item. Then, the warehouse fulfills the order partially. The remaining quantity is fulfilled by the the warehouse holding the next biggest stock of the item. For example, the customer orders 100 tablets. The store's warehouses hold the following stock:
+
+| WAREHOUSE | STOCK |
+|-|-|
+| 2 | 65 |
+| 3 | 40 |
+| 1 | 13 |
+
+In this case, warehouse 2 provides 65 tables by emptying out its stock completely. Warehouse 3 provides the remaining 35 tablets with 5 tablets left in stock.
+
+
+{% info_block warningBox "Item reservation" %}
+Warehouse allocation shouldn't be confused with the reservation process, meaning items are not reserved when they are allocated to a warehouse. The warehouse management system is responsible for warehouse reservation. The default strategy is designed to show how warehouse allocation can be implemented and used in the fulfillment process.
+{% endinfo_block %}
+
+
+## Warehouse picklists
+
+A *warehouse picklist* is a document available in the Fulfillment App that contains a list of items to be picked and shipped to fulfill and order. It includes the following information:
+* Product name
+* Product image
+* Quantity to be picked
+
+Based on a picklist, the warehouse user gathers and prepares the items for shipping.
+
+The information within will also allow you to quickly restock inventory as required.
+
+![screenshot with a piklist]
+
+### Picklist generation strategies
+
+A picklist generation strategies defines how picklists are generated based on the fulfillment requirements of orders per warehouse. On the project level, each warehouse can have its own strategy.
+
+The default picklist generation strategy is designed to generate picklists by order, where each order line is assigned to a unique picklist that contains all the items needed to fulfill the order.
+
+Also, an order can be split into multiple picklists by shipments. If an order contains items that need to be shipped to multiple locations, the order is split into two picklists. This helps the warehouse user ship items more efficiently.
+
+You can also import warehouse strategies by following the (relevant guide)[https://docs.spryker.com/docs/scos/dev/feature-integration-guides/{{page.version}}/install-the-warehouse-picking-feature.html#import-warehouse-picking-list-strategies].
```

Comment:
Besides importing, are there any other ways to define generation strategies?

Reply:
Strategy is defined by developing it, basically we have an extension point where the strategy is defined. Its described in the IG. We can add a reference to it saying that each customer needs to define their own, in the end of the day the one we provide is just an example that is eventually customized to match business needs.